kiali/kiali
# Capture and report request.host information when it is configured

Users have no way to easily see the requested host for their requests. This
can be very useful for failed requests, especially external requests, but is
also has informational value in other cases.

The request.host info must (currently 1.3) be configured manually for Istio, but
if done this PR will capture the information and report it back as part of
the "responses" data, along with rates and response_flags.

UI PR: https://github.com/kiali/kiali-ui/pull/1430
Website PR: https://github.com/kiali/kiali.io/pull/167

Test Instructions:
For this feature to fully work the user needs to do additional configuration of Istio telemetry, adding a field to the istio_requests_total metric.  To begin, test without the customization to ensure things work in the out-of-box scenario.

1. Install the Server and UI PRs.
2. Generate a graph, click an edge, and look at the summary panel.
3. There are now 3 tabs, check the tooltips for the non-default tabs.
4. Click the new Hosts tab.
4.1 It should say something like "Not configured" and direct you to the FAQ
4.2 Install the website PR or just look at the text in the new FAQ file
4.3 Customize the telemetry as described in the FAQ
5. Re-test with the updated telemetry, the Hosts information should now show up.

https://github.com/kiali/kiali/issues/1405
